src/odc/core/MetaData.cc

```diff
@@ -87,26 +88,28 @@ Column* MetaData::columnByName(const std::string& name) const
 bool MetaData::hasColumn(const std::string& name) const
 { 
 	for (size_t i = 0; i < size(); i++)
-		if (at(i)->name() == name || at(i)->name().find(name + "@") == 0)
+		if (columnNameMatches(at(i)->name(), name))
 			return true;
 	return false;
 }
 
 size_t MetaData::columnIndex(const std::string& name) const
 {
-	std::vector<size_t> indices;
+	const size_t notFound = static_cast<size_t>(-1);
```

Comment:
I would prefer not abusing signed/unsigned values here.
Either use a signed integer within the routine (and cast to size_t after the check that it is >= 0), or use a boolean 'found' for the check.